pages/index.html

```diff
@@ -4,9 +4,41 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>a. reyes</title>
+  <link rel="stylesheet" href="/styles/style.css">
+  <link rel="stylesheet" href="/styles/index.css">
 </head>
 <body>
-  
+<section class="page">
```

Comment:
The `<section>` tag is not super well defined, it's intended for "sections of a document", but generally it's used with the `<article>` tag to group text content.  In this case, where you're wrapping the entire page in a section, I recommend pulling the class up to the `<body>` tag and removing the `<section>`:
```
<body class="page">
```